client/my-sites/plans-features-main/index.jsx

```diff
@@ -25,17 +25,15 @@ import {
 	GROUP_JETPACK,
 } from 'lib/plans/constants';
 import { addQueryArgs } from 'lib/url';
+import JetpackFAQ from './jetpack-faq';
```

Comment:
Just so we know those are similar components, I'd probably go for naming them `FaqJetpack` and `FaqWpcom`. Also, should they be `WpcomFaq` and `JetpackFaq` instead of `WpcomFAQ` and `JetpackFAQ`?

Reply:
> Just so we know those are similar components, I'd probably go for naming them `FaqJetpack` and `FaqWpcom`

Yeah, makes sense.

 > Also, should they be `WpcomFaq` and `JetpackFaq` instead of `WpcomFAQ` and `JetpackFAQ`?

I was considering that, but we also still have `FAQ` and `FAQItem` components. I'd opt for consistency here -- even if that's arguably against our naming guidelines -- and possibly do the `FAQ` -> `Faq` renaming in one sweep, i.e. in a separate PR.

Reply:
Ugh, `FAQJetpack` and `FAQWpcom` really look super-ugly. Maybe I'll go with `FaqJetpack` and `FaqWpcom` already...

Reply:
Both of these sound a little weird. Feel free to skip 😆

Reply:
Yeah. F*Jetpcak and F*Wpcom don't sound great...